support/Version.cpp.in

```diff
@@ -0,0 +1,55 @@
+/*
+  Copyright (c) 2022-present, Trail of Bits, Inc.
+  All rights reserved.
+
+  This source code is licensed in accordance with the terms specified in
+  the LICENSE file found in the root directory of this source tree.
+
+  Based on:
+  https://github.com/andrew-hardin/cmake-git-version-tracking/blob/master/better-example/git.cc.in
+  Commit Hash: 98441f9dc056b8d3d4b4287b09e6a99775d70305
```

Comment:
FYI you can combine this info into https://github.com/andrew-hardin/cmake-git-version-tracking/blob/98441f9dc056b8d3d4b4287b09e6a99775d70305/better-example/git.cc.in

There's a an option to `Copy permalink` near the top right of a GitHub file in the `...` dropdown button

Reply:
[03187d0](https://github.com/lifting-bits/sleigh/pull/46/commits/03187d0c589d4aad4523af3dd39bdc5ffa8489bc)